pkg/controller/oneagent/istio.go

```diff
@@ -181,65 +191,93 @@ func (r *ReconcileOneAgent) removeIstioConfigurationForVirtualService(
 	return del
 }
 
-func (r *ReconcileOneAgent) reconcileIstioCreateConfigurations(
-	instance *dynatracev1alpha1.OneAgent,
-	ic *versionedistioclient.Clientset,
-	comHosts []dtclient.CommunicationHost,
-	role string, logger logr.Logger) bool {
+func (r *ReconcileOneAgent) reconcileIstioCreateConfigurations(instance *dynatracev1alpha1.OneAgent,
```

Comment:
Just minor, but having the arguments separated by line, when so many arguments are given, increases readability.

Reply:
since throughout the project the format for declaring multiple vars is the same, it would create some mismatch in the way things are defined in other files. 
We can probably rework this at a later stage, but for now - keeping it as it to maintain uniformity.